### ISSUE TITLE
A question on classifying segments (div types)

### ISSUE THREAD

**Issue by lucagiovannini7** [dracor-org/dracor-schema#43]
Hello, I've seen that in some plays we're using `<castList>` and `<argument>` (= an item summarising the play's plot) as first level divs inside of `<front>`. In the ODD guidelines, however, `<castList>` is nested inside `<div type="Dramatis_Personae">`. As for `<argument>`, I wondered if it was possible to add it to the accepted values for classifying segments. In this way, the structure would be more homogeneous:
```
<front>
  <div type="front"/>
  <div type="prologue"/>
  <div type="Dramatis_Personae"/>
    <castList/>  
  <div type="argument"/>            ### instead of <argument/>
...
</front>
<body>
  <div type="act">
    <div type="scene">
...
</body>
```
What do you think?

**Comment by lehkost** [comment]
Thanks for raising this problem, Luca.

```<div type="Dramatis_Personae"/>``` is a legacy from TextGrid. It doesn't make sense to me to double wrap a cast list when there's a proper TEI element for it. We should get rid of this extra wrapping on the long run and just use ```<castList>``` (in GerDraCor I'm slowly phasing out this unnecessary double wrapping). Same goes for ```<argumentum>```, it's a proper TEI element and we should use it (judging from my encoding practice, this occurs mostly in baroque drama).

```<div type="front"/>``` is another legacy from TextGrid and we should delete this double wrapping, too. ```<front>``` will totally do the job and is also community standard.

Divs within ```<body>``` are a bit trickier in DraCor. ```<div type="prologue">``` could be replaced by ```<prologue>```as well, but we're using divs to divide the play into segments for network extraction, etc.

**Comment by lucagiovannini7** [comment]
Thanks for clarifying. If I understand it correctly, then, divs with `type` attribute are to be employed only for segments inside the `<body>`, while inside `<front>` and `<back>` one should directly use elements like `<castList>` or `<front>` (but also `<preface>`, `<dedication>`, `<postface>`, and so on, I assume). I'll try to stick to this syntax and reformat what I previously did. Perhaps it would be helpful to update the DraCor Oxygen framework with this information, since [the sample file it generates](https://github.com/dracor-org/dracor-oxygen-framework/blob/main/dracor/templates/drama.xml) is often used as a reference for encoding.

**Comment by lehkost** [comment]
Hi Luca, this could be a rule of sorts, but unfortunately there are no such things set in absolute stone yet, historically we developed the schema as we went along, which is difficult with so many different corpora and different literary traditions, there is always something new which asks for inclusion. This is also why the DraCor Oxygen framework is a bit out of step sometimes, sorry for this. Luckily, the TEI standard is stable enough and nothing will really break if we stick to it as good as possible. :)
